repos/system_upgrade/el7toel8/actors/satellite_upgrade_data_migration/actor.py

```diff
@@ -41,3 +43,8 @@ def process(self):
                 os.rmdir(POSTGRESQL_DATA_PATH)
             # move PostgreSQL data to the new home
             shutil.move(POSTGRESQL_SCL_DATA_PATH, POSTGRESQL_DATA_PATH)
```

Comment:
@evgeni  moving the mountpoing leads to the crash of in-place upgrade

Reply:
the same issue happens when you try to remove mountpoint on line 43 if it exists